.travis.yml

```diff
@@ -20,13 +20,12 @@ before_install:
   - conda config --add channels conda-forge
 
 stages:
-  - lint
   - test
 
 stage: test
 
 install:
-  - conda install --file requirements-test.txt
```

Comment:
How come you want to rename this to `requirements.txt`?
We don't need `womtool`, `pytest-workflow`, `wdl-aid`, etc. for the running of the pipeline right? Just for the tests?

Reply:
If I recall correctly, I renamed that file because the version of cromwell in the requirements.txt and requirements-test.txt files got out of sync, which triggered some bug that only happened in one of the cromwell versions.